frontend/lib/tests/rtl-pal.tsx

```diff
@@ -59,7 +64,7 @@ export default class ReactTestingLibraryPal {
       const input = this.rr.getByLabelText(matcher, {
         selector: 'input, select'
       }) as HTMLInputElement;
-      input.value = value;
+      rt.fireEvent.change(input, { target: { value } });
```

Comment:
This is important because it causes the `onChange` handlers of the input fields to be called, if they are set, which is particularly important for the autocomplete widget. I actually had to get assistance from the Downshift community to [figure this out](https://github.com/paypal/downshift/issues/583) and they were very quick to help out.